source/developers-guide/attribute-system/index.md

```diff
@@ -0,0 +1,654 @@
+---
+layout: default
+title: Attribute system
+github_link: developers-guide/attribute-system/index.md
+shopware_version: 5.2.0
+indexed: true
+tags:
+  - attributes
+---
+
+The attribute system allows developers and users to configure additional fields for different entities in shopware. Users can simply define new fields over the free text fields backend module.
+
+Developers can define new attributes over the database or use the corresponding service `Shopware\Bundle\AttributeBundle\Service\CrudService`.
+
+# Services
```

Comment:
All headers must be changed one level deeper:

`# Services -> ## Services'

The Title is used as h1 automatically.